build.gradle

```diff
@@ -23,6 +23,7 @@ ext {
     jackson_version = '2.8.3'
     care4alfVersion = '2.3.0'
     http_version = '4.3.4' // Used by integration tests
+    ootbee_version = '1.2.2.0' // Used in docker image for testing
```

Comment:
Since this is only used once, why not keep it locally, close to where it is used? It makes development easier since there is less switching needed.